script.js

```diff
@@ -843,15 +843,15 @@ document.addEventListener('DOMContentLoaded', function() {
       }
     });
     if (date.getHours() != 0 && date.getHours() < 11 && date.getHours() > 3) {
-      document.querySelector('#overlayGreeting').innerText = (core.stg.lang == 'en') ? 'Good morning!' : 'Nangāle!';
+      document.querySelector('#overlayGreeting').innerText = (core.stg.lang == 'en') ? 'Good morning!' : (core.stg.lang == 'un') ? 'Nammadhyāno(m)!' : 'शुभ प्रभात';
```

Comment:
so... the morning is also `'Nammadhyāno(m)!'`?

Reply:
Bruh no I forgot that
It’s nangāle

Reply:
ok.. I'll fix it myself than
thanks:)